src/components/shared/avatarWithChip/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Stack from '@mui/material/Stack';
+
+import BasicAvatar from '@/components/shared/avatar';
+import BasicChip from '@/components/shared/chip';
+import BasicAvatarProps from '@/types/components/BasicAvatarProps';
+
+interface Props extends BasicAvatarProps {
+  label: string;
+}
+
+const AvatarWithChip = ({
+  label,
+  size = 60,
+  imgSrc,
+  alt,
+  isUserOn = false,
+  onClick,
+}: Props) => {
+  return (
+    <Stack width={size} spacing={0.5}>
+      <BasicAvatar
+        size={size}
+        imgSrc={imgSrc}
+        alt={alt}
+        isUserOn={isUserOn}
+        onClick={onClick}
+      />
+      <BasicChip label={label} size="small" />
+    </Stack>
+  );
+};
```

Comment:
약간 이런 느낌으로 수정해볼 수도 있지 않을까요?

```suggestion
const AvatarWithChip = ({
  size = 60,
  isUserOn = false,
  ...props
}: Props) => {
  return (
    <Stack width={size} spacing={0.5}>
      <BasicAvatar
        {...props}
      />
      <BasicChip label={label} size="small" />
    </Stack>
  );
};
```

Reply:
prps 정리하니깐 깔끔하네요

Reply:
아, 저는 코드리뷰하면서 들었던 제 생각을 제안했던 것인데요,

아무래도 props 의 개수가 많아질수록 코드가 읽기 힘들어지기 때문에
타입추론으로 파악할 수 있는 속성들은 spread 를 이용해서 받아주거나, 꼭 명세가 필요한 속성들은 객체로 만들어 사용하는 편이 코드가 좀 더 깔끔해질 것 같다고 생각했기 때문이었습니다ㅎㅎ

원지님께서, 컴포넌트를 사용하는 위치에서 바로바로 볼 수 있게 명세하려는 의도였다면, 저는 지금 코드도 충분히 좋다고 생각합니다ㅎㅎ